inbm/fpm/inb-provision-cloud/main.go

```diff
@@ -255,21 +257,39 @@ func configureUcc(cloudCredentialDir string, templateDir string) string {
 		jsonTemplate = createUnencryptedTemplate(templateDir)
 	}
 
-	clientId := getClientId()
+	clientId := getIdFromFile(uccClientIdFile)
+	serverId := getIdFromFile(uccServerIdFile)
+	if !isIdValid(serverId) {
```

Comment:
We should also be checking for ClientID file.
I do see isIdValid() checks for uuid as well as ip, so should be able to support clientID checks as well.

Reply:
@pillaiabhishek - I'm not sure what they want to validation on the client ID.  I'm afraid to limit it and then it limits what they want to do.  I believe they said something about a mac address, but I'm not 100% and they could have changed their minds.

Reply:
I'm going to go with this format: 
Topic names must include at least one character and can consist of any UTF-8 characters, except '$,' '*,' '+,' and '#. ' '$' denote sys topics used by MQTT brokers to publish data on broker performance. '+' and '#' are single and multi-level